containers/integration-test-base

```diff
@@ -12,5 +12,6 @@ RUN dnf upgrade --refresh --nodocs -y && \
             systemd \
             systemd-devel \
             valgrind \
+            bluechi-is-online \
```

Comment:
Please remove this line. 
Any BlueChi component is only installed in the local or snapshot container image.

Reply:
Done.